extensions/cl_intel_subgroups_short.asciidoc

```diff
@@ -533,10 +549,11 @@ None.
 [grid="rows"]
 [options="header"]
 |========================================
-|Rev|Date|Author|Changes
-|1|2016-10-20|Ben Ashbaugh|*First public revision.*
-|2|2018-11-15|Ben Ashbaugh|Conversion to asciidoc.
-|3|2019-09-17|Ben Ashbaugh|Added vec3 types for shuffles and asciidoctor formatting fixes.
+|Version|Date|Author|Changes
+|rev 1|2016-10-20|Ben Ashbaugh|*First public revision.*
+|rev 2|2018-11-15|Ben Ashbaugh|Conversion to asciidoc.
+|rev 3|2019-09-17|Ben Ashbaugh|Added vec3 types for shuffles and asciidoctor formatting fixes.

```

Comment:
Do you want to specify what version gets reported for rev1-3? I guess there are implementations in the field that don't support versioning at all though.

Reply:
Good point - I added a line to the table to say that v1.0.0 is equivalent to rev3.